test/jdk/javax/swing/JColorChooser/Test6977726.java

```diff
@@ -21,22 +21,40 @@
  * questions.
  */
 
+import javax.swing.JColorChooser;
+import javax.swing.JLabel;
+
 /*
  * @test
  * @bug 6977726
- * @summary Checks if JColorChooser.setPreviewPanel removes the old one
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test6977726.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Checks if JColorChooser.setPreviewPanel(JLabel) doesn't remove the preview panel but
+ *          removes the content of the default preview panel
+ * @run main/manual Test6977726
  */
 
-import javax.swing.JApplet;
-import javax.swing.JColorChooser;
-import javax.swing.JLabel;
+public class Test6977726 {
+
+    public static void main(String[] args) throws Exception {
+        String instructions = """
+                Check that there is a panel with \"Text Preview Panel\" text 
+                and with title \"Preview\" in the JColorChooser.""";
```

Comment:
(1) You have a whitespace error that needs fixing.
(2) It is probably worth adding a sentence "Test passes if the panel is as described, test fails otherwise".

Reply:
@alisenchung Since you are using string block there is no need to escape double-quotes.